src/shogun/evaluation/MachineEvaluation.cpp

```diff
@@ -121,5 +165,19 @@ CMachine* CMachineEvaluation::get_machine() const
 
 EEvaluationDirection CMachineEvaluation::get_evaluation_direction()
 {
-	return m_evaluation_criterion->get_evaluation_direction();
+    REQUIRE(m_list_evaluation_criterion->get_array_size()==1,"Multiple Metrics provided. Please use get_evaluation_directions()");
```

Comment:
Pls check how we do whitespace formatting in Shogun
We also have a charater limit per line